cmd/zc_traverser_file.go

```diff
@@ -67,6 +70,10 @@ func (t *fileTraverser) Traverse(preprocessor objectMorpher, processor objectPro
 
 	// if not pointing to a share, check if we are pointing to a single file
 	if targetURLParts.DirectoryOrFilePath != "" {
+		if t.trailingDot != common.ETrailingDotOption.Enable() && strings.HasSuffix(targetURLParts.DirectoryOrFilePath, ".") {
```

Comment:
Ditto

Reply:
removed the nil but I will leave the log here as a warning